src/models/image-map.ts

```diff
@@ -53,6 +52,7 @@ export interface IImageHandlerStoreResult {
 }
 interface IImageMap {
   unitCodeMap?: IMSTMap<ISimpleType<string>>;
+  unitUrl?: string;
```

Comment:
If you make my change below so that the `localAssetsImagesHandler` only uses the `curriculumUrl`, then I think you can remove `unitUrl` from this interface.